src/Nethermind/Nethermind.Db/IPruningConfig.cs

```diff
@@ -79,4 +79,7 @@ The max number of parallel tasks that can be used by full pruning.
 
     [ConfigItem(Description = "The number of past states before the state gets pruned. Used to determine how old of a state to keep from the head.", DefaultValue = "64")]
     int PruningBoundary { get; set; }
+
+    [ConfigItem(Description = "Dirty node shard count", DefaultValue = "8")]
+    int DirtyNodeShardBit { get; set; }
```

Comment:
Should it default to something based on CPU count maybe?

Reply:
Maybe, one day. But for now I can't find any difference unless it is set too low where pruning can't parallelize.